dial_test.go

```diff
@@ -29,31 +32,37 @@ func TestDialer_Dial_error(t *testing.T) {
 		err: errors.New(errMsg),
 	}
 
-	conn, err := tarantool.Connect("any", tarantool.Opts{
-		Dialer: dialer,
-	})
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
+	conn, err := tarantool.Connect(ctx, "any",
+		tarantool.Opts{
+			Dialer: dialer,
+		})
```

Comment:
Just a codestyle, I don't see reasons for the change.
```suggestion
	conn, err := tarantool.Connect(ctx, "any",tarantool.Opts{
		Dialer: dialer,
	})
```

Reply:
Updated.